proto/cosmos/group/v1/module/v1/module.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package cosmos.group.v1.module.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+
+// Module is the config object of the group module.
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
+    go_import: "github.com/cosmos/cosmos-sdk/x/group"
+  };
+
+  // group_config is the config for group module.
+  Config group_config = 1;
```

Comment:
we could also just put max_execution_period and max_metadata_len directly in there without using some addition `Config` (just as in auth module.proto)